cluster/gce/util.sh

```diff
@@ -49,7 +56,7 @@ function get_latest_gci_image() {
   echo "${gci_images[0]}"
 }
 
-if [[ "${OS_DISTRIBUTION}" == "gci" ]]; then
+if [[ "${MASTER_OS_DISTRIBUTION}" == "gci" ]]; then
   # If the master image is not set, we use the pinned GCI image.
```

Comment:
`we used the pinned GCI image` isn't true any more, now that you do `get_latest_gci_image`.